Repository/PhpcrRepository.php

```diff
@@ -119,4 +122,36 @@ protected function buildCollection(array $nodes)
 
         return $collection;
     }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function removeNodes(array $nodes)
+    {
+        foreach ($nodes as $node) {
+            $node->remove();
+        }
+
+        $this->session->save();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function moveNodes(array $nodes, $sourceQuery, $targetPath)
+    {
+        $this->doMoveNodes($nodes, $sourceQuery, $targetPath);
+        $this->session->save();
+    }
+
+    private function doMoveNodes(array $nodes, $sourceQuery, $targetPath)
+    {
+        if (false === $this->isGlobbed($sourceQuery)) {
```

Comment:
if it's not globbed, wouldn't it not always result in `$nodes` containing just one element? In other words, why can't we simply loop through `$nodes` like done in `removeNodes()`?